verifier/verifier.py

```diff
@@ -29,6 +29,9 @@ def __init__(self):
     # All the items that will be verified
     self.verify_plans = []
 
+    # Filename used for the json version of verifier output
+    self.report_filename = 'verifier_test_report.json'
```

Comment:
suggestion: convert this from a field to a global constant b/c I don't think you expect this to change from instance to the next at runtime